src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -76,9 +76,6 @@ export function CurrencySearch({
   const searchTokenIsAdded = useIsUserAddedToken(searchToken)
 
   const defaultTokens = useDefaultActiveTokens(chainId)
-  const filteredTokens: Token[] = useMemo(() => {
```

Comment:
i just moved this into the `sortedTokens` memo function since it's only used once.